datawig/simple_imputer.py

```diff
@@ -563,15 +563,16 @@ def load_hpo_model(self, hpo_name: int = None):
         """
 
         if self.hpo.results is None:
-            logger.warn('No hpo run available. Run hpo calling SimpleImputer.fit_hpo().')
+            logger.warn('No hpo run available. Run hpo by calling SimpleImputer.fit_hpo().')
             return
 
         if hpo_name is None:
             if self.output_type == 'numeric':
-                hpo_name = self.hpo.results['mse'].astype(float).idxmax()
+                hpo_name = self.hpo.results['mse'].astype(float).idxmin()
```

Comment:
great, thanks! did we also fix the metrics now so that the right metric name is logged when optimizing?

Reply:
I didn't change how metrics are logged.